### PR TITLE
validate swagger endpoints

### PR DESCRIPTION
**Pull request by sullis**
- validate swagger json endpoint
- validate swagger yaml endpoint

### CODE REVIEW

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/AdminConsoleIntegrationTest.java [hunk 1]
```diff
@@ -90,4 +91,29 @@ public void testApiHelp()
...
+  public void testSwaggerJson(final String description, final String endpointBase)
+      throws Exception {
+    String response = sendGetRequest(endpointBase + "/swagger.json");
+    Assert.assertTrue(response.startsWith("{\"swagger\":\"2.0\""));
+    Assert.assertTrue(response.endsWith("}"));
+  }
+
+  @DataProvider
+  public Object[][] endpointBase() {
+    return new Object[][]{
```
Comment (Jackie-Jiang) [inline pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/AdminConsoleIntegrationTest.java:112]: (format) Please reformat this part

**Review by Jackie-Jiang — APPROVED** [review]

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/AdminConsoleIntegrationTest.java (hunk 1 above)
Comment (sullis) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/AdminConsoleIntegrationTest.java]: Done

**Comment by sullis** [comment]
ready to merge

**Comment by sullis** [comment]
@Jackie-Jiang ready to merge

**Pull request merged**